source/extensions/filters/common/ext_authz/ext_authz_http_impl.h

```diff
@@ -98,6 +98,14 @@ class ClientConfig {
    */
   const MatcherSharedPtr& upstreamHeaderMatchers() const { return upstream_header_matchers_; }
 
+  /**
+   * Returns a list of matchers used for selecting the authorization response headers that
+   * will be append when using same key should be send to an the upstream server.
```

Comment:
Nit. will be appended when using the same key should be sent to the upstream server.